tutorpod_autoscaling/templates/pod-autoscaling/k8s/keda.yml

```diff
@@ -0,0 +1,23 @@
+{% for worker, autoscaling in POD_AUTOSCALING_KEDA_CONFIG.items() %}
+---
+apiVersion: keda.sh/v1alpha1
+kind: Keda
```

Comment:
What do we need to install to get the Keda CRD available?

Reply:
[This](https://github.com/eduNEXT/manifests-template/pull/50) answers my question